chromium_src/content/browser/media/cdm_registry_impl.cc

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "content/browser/media/cdm_registry_impl.h"
+
+#if defined(OS_LINUX)
+#include <algorithm>
+#endif
+
+#define Init Init_ChromiumImpl
+#include "../../../../../content/browser/media/cdm_registry_impl.cc"
+#undef Init
+
+namespace content {
+void CdmRegistryImpl::Init() {
+  Init_ChromiumImpl();
+
+#if defined(OS_LINUX)
```

Comment:
I think you're missing what I was trying to say here. It calls `GetContentClient()->AddContentDecryptionModules` and we can implement `AddContentDecryptionModules` in `BraveContentClient`

Reply:
I thought that first.
But, other places needs original list like `PreloadLibraryCdms()` in `content_main_runner_impl.cc`.

Reply:
I don't understand, the result would be identical because it passes a reference that can be modified

Reply:
`ChromeContentClient::AddContentDecryptionModules()` fills cdms to passed pointer.
Caller of above method will have different instance.
Of course, client of `CdmRegistry` will see same list.

Reply:
Oh, many comments were deleted..
Caller of `AddContentDecryptionMoudles` maintains it's own cmd list.
Clients of `CdmRegistry` see same list because `CdmRegistry::GetAllRegisteredCdms()` returns list reference.